frontend/scripts/utils/getURLFromParams.js

```diff
@@ -130,7 +131,7 @@ const API_ENDPOINTS = {
   },
   [GET_PROFILE_METADATA]: {
     api: 3,
-    endpoint: '/profiles/$node_id$/profile_meta'
+    endpoint: '/contexts/$context_id$/nodes/$node_id$/profile_metadata'
```

Comment:
this is the old url, it will redirect to the new one. please use the new one with ?context_id=xx or ?commodity_id=xx so that we can retire the redirect once not needed anymore.